### PR TITLE
38 - Add fetch query to get available genders

### PR DESCRIPTION
**Pull request by dainius-jankunas**

### CODE REVIEW

**Review by ignasposka — APPROVED** [review]
Looks good!

File: src/schemas/gender.ts [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+import getGendersQuery from '../sql-queries/gender';
+
+const typeDef = `
+extend type Query {
+  """
+    Get all genders.
+  
+    Examples:
+  
+    genders(language: 'lt')
```
Comment (AzisK) [inline src/schemas/gender.ts:11]: Single quoted `'lt'` gives me an error. It is only happy with double quotes `"lt"`
```
{
  genders(language: "lt") {
    id
  }
}
```

File: src/sql-queries/gender.ts [hunk 2]
```diff
@@ -0,0 +1,19 @@
+import { QueryConfig } from 'pg';
+
+const getGendersQuery = (language: string): QueryConfig => {
+    const text = `SELECT 
+                    gender as id, 
+                    translation as value
+                FROM gender_translation
+                WHERE gender = ANY(enum_range(null::gender))
```
Comment (AzisK) [inline src/sql-queries/gender.ts:8]: Do we need this WHERE? Enum itself allows only enum values inside, so this additional check is not needed, I would say

**Review by AzisK — CHANGES_REQUESTED** [review]
Looking good overall. Left a few comments for improvements

File: src/schemas/gender.ts (hunk 1 above)
Comment (dainius-jankunas) [reply src/schemas/gender.ts:11]: fixing that

File: src/sql-queries/gender.ts (hunk 2 above)
Comment (dainius-jankunas) [reply src/sql-queries/gender.ts:8]: will remove it

**Review by JustasBelevicius — APPROVED** [review]

File: src/sql-queries/gender.ts [hunk 3]
```diff
@@ -0,0 +1,19 @@
+import { QueryConfig } from 'pg';
+
+const getGendersQuery = (language: string): QueryConfig => {
+    const text = `SELECT 
+                    gender as id, 
+                    translation as value
+                FROM gender_translation
+                WHERE language = $1
+                ORDER BY id;`;
```
Comment (AzisK) [inline src/sql-queries/gender.ts:9]: We did not order Animals, so I suppose we should not order Genders as well

**Review by AzisK — APPROVED** [review]
I suppose we should not order genders since we did not order animals. Everything else looks smooth

**Pull request merged**